docs/conf.py

```diff
@@ -77,11 +74,13 @@
 }
 html_baseurl = "https://sphinx-book-theme.readthedocs.io/en/latest/"
 
+# -- Custom configuration ---------------------------------------------------
+
+
+# Always regenerate the index page, to ensure assets are correctly copied.
+def env_get_outdated(app, env, added, changed, removed):
+    return ["index"]
+
 
-# -- Custom scripts ----------------------------------------------------------
-# Grab the latest contributing docs
-url_contributing = (
```

Comment:
where are we now grabbing this file? Or did it get removed entirely?

Reply:
yep removed and just link to https://executablebooks.org/en/latest/contributing.html
I don't think we want to be cloning it in every repo, because otherwise if/when it gets altered, it means we have to remember to rebuild every repo's documentation

Reply:
makes sense - maybe what we should do is have it as an external link in our table of contents. I'm just worried about people not noticing it if it is "just" a link in our docs. Either putting it in TOC, or starting our contributions page with a big admonition or something so that people see it?

Reply:
Possibly, but it is mentioned in quite a few places now: theres a link to it at the bottom of this and all issue/PR pages, and the welcomebot mentions it for first time issues and PRs

Reply:
and github also shows a welcome box with it in the first time you go to open an issue in a new repo

Reply:
yeah - I guess we can see if it is an issue for people noticing it. My assumption is always that readers of documentation have 2-second attention spans and won't notice anything unless in giant and/or colored/bolded letters :-)

Reply:
I'm gonna say this is resolved, for now. :)